test/python/test_nixl_api.py

```
@@ -223,6 +223,39 @@ def test_improper_get_reg_descs(one_empty_agent, one_xfer_list):
+    try:
+        reg1 = agent1.get_reg_descs([(addr1, size, 0, "")], mem_type="DRAM")
+        reg2 = agent2.get_reg_descs([(addr2, size, 0, "")], mem_type="DRAM")
+        agent1.register_memory(reg1)
+        agent2.register_memory(reg2)
+
+        # Local overload: pass a nixlXferDList describing agent1's own buffer.
+        local_xfer = agent1.get_xfer_descs(
+            [(addr1, size, 0)], mem_type="DRAM"
+        )
+        local_mvh = agent1.prep_mem_view(local_xfer)
+        assert isinstance(local_mvh, int) and local_mvh != 0
```

Ruff (0.15.15)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)